org.flatpak.Builder.yml

```
@@ -99,6 +99,11 @@ modules:
       - type: archive
         url: https://github.com/ostreedev/ostree/releases/download/v2020.3/libostree-2020.3.tar.xz
         sha256: 5877396ea092f5b6701c17bd20280746cad20f55a94c760dbeba0fa115818c05
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/ostreedev/ostree/releases/latest
+          version-query: "first | .tag_name"
+          url-query: 'first | .assets | first | .browser_download_url'
```

It's a single latest release, so you don't need to pull first from array:
```suggestion
          url: https://api.github.com/repos/ostreedev/ostree/releases/latest
          version-query: ".tag_name"
          url-query: '.assets | first | .browser_download_url'
```